notebooks/MagneticLattices.ipynb

```diff
@@ -613,53 +624,83 @@
     "openfermion_hamiltonian_kitaev = pyliqtr_hamiltonian_to_openfermion_qubit_operator(H_kitaev)\n",
     "openfermion_hamiltonian_directional_triangle = pyliqtr_hamiltonian_to_openfermion_qubit_operator(H_directional_triangle)\n",
     "\n",
-    "trotter_square_metadata = EstimateMetaData(\n",
+    "#Explicitly set the trotter order to 2, is set to 2 by default by estimate.\n",
+    "\n",
+    "# defining precision required for the trotterized circuit\n",
+    "energy_precision = 1e-6\n",
+    "evolution_time=1000\n",
+    "\n",
+    "trotter_order=2\n",
+    "\n",
+    "extrapolate=True\n",
+    "\n",
+    "trotter_square_metadata = TrotterMetaData(\n",
     "    id=uid,\n",
     "    name='square_lattice_trotter',\n",
     "    category='scientific',\n",
     "    size=f'lattice_size: {square_lattice_size}',\n",
     "    task='Time Dependent Dynamics',\n",
-    "    implementations=f'trotterization, JT={evolution_time}, numsteps={numsteps}, required_precision={required_precision}',\n",
+    "\n",
+    "    evolution_time=evolution_time,\n",
+    "    trotter_order= 2,\n",
+    "    is_extrapolated=extrapolate,\n",
+    "    energy_precision=required_precision,\n",
     ")\n",
     "uid += 1\n",
     "\n",
-    "trotter_triangle_metadata = EstimateMetaData(\n",
+    "trotter_triangle_metadata = TrotterMetaData(\n",
     "    id=uid,\n",
     "    name='triangle_lattice_trotter',\n",
     "    category='scientific',\n",
-    "    size=f'lattice_size: {triangle_lattice_size}',\n",
+    "    size=f'lattice_size: {square_lattice_size}',\n",
     "    task='Time Dependent Dynamics',\n",
-    "    implementations=f'trotterization, JT={evolution_time}, numsteps={numsteps}, required_precision={required_precision}',\n",
+    "\n",
+    "    evolution_time=evolution_time,\n",
+    "    trotter_order= 2,\n",
+    "    is_extrapolated=extrapolate,\n",
+    "    energy_precision=required_precision,\n",
     ")\n",
     "uid += 1\n",
     "\n",
-    "trotter_cubic_metadata = EstimateMetaData(\n",
+    "trotter_cubic_metadata = TrotterMetaData(\n",
     "    id=uid,\n",
-    "    name='triangle_lattice_trotter',\n",
+    "    name='cubic_lattice_trotter',\n",
     "    category='scientific',\n",
-    "    size=f'lattice_size: {cubic_lattice_size}',\n",
+    "    size=f'lattice_size: {square_lattice_size}',\n",
```

Comment:
any reason to use square_lattice_size instead of cubic_lattice_size? Same for Triangle.

Reply:
This should be fixed. It was found in an earlier review so Im guessing George missed it